src/flash/display/DisplayObjectContainer.ts

```diff
@@ -94,6 +94,7 @@ module Shumway.AVM2.AS.flash.display {
           child.dispatchEvent(events.Event.getInstance(events.Event.AVM1_INIT));
           child.dispatchEvent(events.Event.getInstance(events.Event.AVM1_CONSTRUCT));
           child._setFlags(DisplayObjectFlags.NeedsLoadEvent);
+          this._setFlags(DisplayObjectFlags.ContainsFrameScriptPendingChildren);
```

Comment:
You're right, except test should, I think, be `child._hasAnyFlags(DisplayObjectFlags.HasFrameScriptPending | DisplayObjectFlags.ContainsFrameScriptPendingChildren)`. I added that.